### ISSUE TITLE
LSP Completion an script that extends another script crashs Godot

### ISSUE THREAD

**Issue by rafaeldelboni** [godotengine/godot#36680]
**Godot version:**
3.2 branch with this https://github.com/godotengine/godot/pull/36521

**OS/device including version:**
Arch Linux 5.5.5

**Issue description:**
LSP crashes Godot when a script inherits another script.

**Steps to reproduce:**
Create two scripts and make one inherits another (character -> player), open the base script (character) and try auto-completion over LSP.

**Backtrace**
```
handle_crash: Program crashed with signal 11
Dumping the backtrace. Please include this when reporting the bug on https://github.com/godotengine/godot/issues
[1] /usr/lib/libc.so.6(+0x3bd70) [0x7fbcacedad70] (??:0)
[2] Ref<SceneState>::operator->() const (/home/delboni/Workspace/godot/./core/reference.h:117)
[3] PackedScene::instance(PackedScene::GenEditState) const (/home/delboni/Workspace/godot/scene/resources/packed_scene.cpp:1695)
[4] GDScriptWorkspace::_get_owner_node(String) (/home/delboni/Workspace/godot/modules/gdscript/language_server/gdscript_workspace.cpp:416)
[5] GDScriptWorkspace::completion(lsp::CompletionParams const&, List<ScriptCodeCompletionOption, DefaultAllocator>*) (/home/delboni/Workspace/godot/modules/gdscript/language_server/gdscript_workspace.cpp:430 (discriminator 1))
[6] GDScriptTextDocument::completion(Dictionary const&) (/home/delboni/Workspace/godot/modules/gdscript/language_server/gdscript_text_document.cpp:152 (discriminator 3))
[7] MethodBind1R<Array, Dictionary const&>::call(Object*, Variant const**, int, Callable::CallError&) (/home/delboni/Workspace/godot/./core/method_bind.gen.inc:961 (discriminator 9))
[8] Object::call(StringName const&, Variant const**, int, Callable::CallError&) (/home/delboni/Workspace/godot/core/object.cpp:903 (discriminator 1))
[9] Object::callv(StringName const&, Array const&) (/home/delboni/Workspace/godot/core/object.cpp:811)
[10] JSONRPC::process_action(Variant const&, bool) (/home/delboni/Workspace/godot/modules/jsonrpc/jsonrpc.cpp:125 (discriminator 1))
[11] JSONRPC::process_string(String const&) (/home/delboni/Workspace/godot/modules/jsonrpc/jsonrpc.cpp:160 (discriminator 1))
[12] GDScriptLanguageProtocol::process_message(String const&) (/home/delboni/Workspace/godot/modules/gdscript/language_server/gdscript_language_protocol.cpp:140)
[13] GDScriptLanguageProtocol::LSPeer::handle_data() (/home/delboni/Workspace/godot/modules/gdscript/language_server/gdscript_language_protocol.cpp:95)
[14] GDScriptLanguageProtocol::poll() (/home/delboni/Workspace/godot/modules/gdscript/language_server/gdscript_language_protocol.cpp:243)
[15] GDScriptLanguageServer::_notification(int) (/home/delboni/Workspace/godot/modules/gdscript/language_server/gdscript_language_server.cpp:63)
[16] GDScriptLanguageServer::_notificationv(int, bool) (/home/delboni/Workspace/godot/modules/gdscript/language_server/gdscript_language_server.h:39 (discriminator 14))
[17] Object::notification(int, bool) (/home/delboni/Workspace/godot/core/object.cpp:915)
[18] SceneTree::_notify_group_pause(StringName const&, int) (/home/delboni/Workspace/godot/scene/main/scene_tree.cpp:988)
[19] SceneTree::idle(float) (/home/delboni/Workspace/godot/scene/main/scene_tree.cpp:526 (discriminator 3))
[20] Main::iteration() (/home/delboni/Workspace/godot/main/main.cpp:2062)
[21] OS_X11::run() (/home/delboni/Workspace/godot/platform/x11/os_x11.cpp:3321)
[22] /home/delboni/Workspace/godot/bin/godot.x11.tools.64(main+0x121) [0x16fa8c7] (/home/delboni/Workspace/godot/platform/x11/godot_x11.cpp:57)
[23] /usr/lib/libc.so.6(__libc_start_main+0xf3) [0x7fbcacec6023] (??:0)
[24] /home/delboni/Workspace/godot/bin/godot.x11.tools.64(_start+0x2e) [0x16fa6ee] (??:?)
-- END OF BACKTRACE --

```

**Comment by rafaeldelboni** [comment]
@Geequlim I'm over this, I just discovered the root cause.

When a script inherits another it shows the other script as owner, so I will filter out from the owners list only the scenes, that maybe fix this issue.

Sorry about that.
![2020-02-29_15:18:55](https://user-images.githubusercontent.com/1683898/75613054-30835200-5b08-11ea-871e-9e0d8a37abb5.png)

**Comment by rafaeldelboni** [comment]
This should be fixed with #36683 sorry again for the trouble.

**Comment by Geequlim** [comment]
Any backtrace log for the crash ?
